### ISSUE TITLE
Please HELP: Insert key on OS X with a PC keyboard

### ISSUE THREAD

**Issue by psprint** [milkytracker/MilkyTracker#176]
I need to insert blank notes into the current track, to move notes around, together with the Backspace key (which moves the notes 1 line up). However, on OS X, the Insert key seems to be **unused**. Various posts say that on OS X, they key F13 is being mapped to `Insert note` instead of the Insert key.

However, I'm using a PC keyboard with my Mac, and I don't have the F13 key... **Please help, how to insert notes into a track on OS X?**

**Comment by SyntaxErrol** [comment]
The system menu at the top of the screen used to have a couple of options for Insert mapping on Mac. Maybe they're still there.

**Comment by yzimsx** [comment]
On an external PC keyboard, the PrintScreen key seemed to work as Insert. There's no F13 key on a MacBook Pro, so you can't Insert in MilkyTracker straight out of the box.

**Comment by Deltafire** [comment]
Commit d796a98 maps alt-backspace to "Insert space on current track at cursor position" and shift-alt-backspace to "Insert row at cursor position".

**Comment by psprint** [comment]
Great, thanks!

**Comment by yzimsx** [comment]
I couldn't get Insert working even in the latest master version, so I changed the F12 to Insert in src/tracker/cocoa/MTKeyTranslator.mm
like this:
		case kVK_F12:			return VK_INSERT;

I don't understand how the Insert key was supposed to work. There's just one backspace key on MacBooks - how is it supposed to work as Backspace, Delete _and_ Insert keys, and all of their modifier combinations? I tried all possible modifier key combinations but couldn't get it to work as Insert.
